lib/ActivityStreams.js

```diff
@@ -60,6 +61,17 @@ function ActivityStreams(options = {}) {
   this._memoized = this._get_memoized(this._memoizer);
 
   this._telemetrySender = new TelemetrySender();
+
+  this._experimentData = {};
+  let rng = seedrandom(options.clientID);
+  // Targeting 10% of users for this experiment
+  if (rng() <= 0.1) {
+    this._experimentData.id = "exp-001-delete-menu";
+    this._experimentData.reverseMenuOptions = true;
```

Comment:
I would recommend using string or integer based variant IDs such as `"CONTROL"` and `"REVERSE"`.  Also, you might want to call the experiment something neutral like `deleteMenuOrder`